config/locales/devise/en.yml

```diff
@@ -142,18 +142,18 @@ en:
         Your phone number has been set. Confirm it by entering the passcode below.
       please_try_again_html: Please try again in %{time_remaining}.
       recovery_code_fallback:
-        link: personal key
-        text_html: Don’t have access to your phone? Use a %{link} instead.
+        link: Use a personal key instead
+        text_html: Don’t have access to your phone? %{link}.
       recovery_code_header_text: Enter your personal key
       recovery_code_prompt: >
         You can use this personal key once. If you still need a code after
         signing in, go to your account settings page to get a new one.
       totp_fallback:
-        sms_link_text: receive a code via text message
+        sms_link_text: get a code via text message
         text_html: >
           If you can’t use your authenticator app right now you can %{sms_link}
           or %{voice_link}.
-        voice_link_text: with a phone call
+        voice_link_text_html: <span class='hide'>get a code</span> with a phone call
```

Comment:
@jessieay I updated the PR and see the benefit of having the HTML out of the i18n files, but having four separate translation strings for a single sentence also seems like it would be confusing for someone trying to piece together the overall context of the content down the line.  It may not be apparent from just looking at the yaml file that some of the text is hidden either unless we decide on using a consistent naming convention like `sr_only`.